homeassistant/components/qbittorrent/const.py

```diff
@@ -0,0 +1,6 @@
+"""Constants for qBittorrent."""
+
+DEFAULT_NAME = "qBittorrent"
+SENSOR_TYPE_CURRENT_STATUS = "current_status"
+SENSOR_TYPE_DOWNLOAD_SPEED = "download_speed"
+SENSOR_TYPE_UPLOAD_SPEED = "upload_speed"
```

Comment:
`SENSOR_TYPE_***` implies that it is really specific to the sensor platform. I'm not so sure they should be moved out.

`DEFAULT_NAME` makes sense.